.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -13,4 +13,4 @@ dependencies:
   - pip:
     - future
     - scipy
-    - av
```

Comment:
Should this be `!= 10.0` or something like that, if we expect a bugfix release on the pyav side?